config.js

```diff
@@ -882,6 +882,8 @@ config.NSFS_LOW_FREE_SPACE_PERCENT_UNLEASH = 0.10;
 // anonymous account name
 config.ANONYMOUS_ACCOUNT_NAME = 'anonymous';
 
+config.NSFS_UPLOAD_STREAM_MAX_BUFFER_SIZE = 64 * 1024 * 1024;
```

Comment:
I am not sure if this is too much. The default in nodejs is 64KB and we do want to increase but maybe not so much. Perhaps it would be best that we set the default to 8MB and if needed we can instruct users how to increase further.

Also I think the name I used ..._MAX_BUFFER_SIZE might be thought to be something else so better rename to something like NFSF_UPLOAD_STREAM_MEM_THRESHOLD

see https://nodejs.org/docs/latest-v20.x/api/stream.html#buffering